shogun/classifier/svm/SVMSGD.cpp

```diff
@@ -214,13 +213,16 @@ void CSVMSGD::init()
 	loss=new CHingeLoss();
 	SG_REF(loss);
 
-    m_parameters->add(&C1, "C1",  "Cost constant 1.");
-    m_parameters->add(&C2, "C2",  "Cost constant 2.");
-    m_parameters->add(&wscale, "wscale",  "W scale");
-    m_parameters->add(&bscale, "bscale",  "b scale");
-    m_parameters->add(&epochs, "epochs",  "epochs");
-    m_parameters->add(&skip, "skip",  "skip");
-    m_parameters->add(&count, "count",  "count");
-    m_parameters->add(&use_bias, "use_bias",  "Indicates if bias is used.");
-    m_parameters->add(&use_regularized_bias, "use_regularized_bias",  "Indicates if bias is regularized.");
+	SG_ADD(&C1, "C1", "Cost constant 1.", MS_AVAILABLE);
+	SG_ADD(&C2, "C2", "Cost constant 2.", MS_AVAILABLE);
+	SG_ADD(&wscale, "wscale", "W scale", MS_NOT_AVAILABLE);
```

Comment:
are these parameters necessary to be accessible from the outside/equals/serilazation, or are they only to store the intermediate state of training?